src/Bundle/Resources/config/services.xml

```diff
@@ -37,13 +37,15 @@
         <defaults public="true" />
 
         <service id="sylius.random_generator" class="Sylius\Component\Resource\Generator\RandomnessGenerator" />
+        <service id="Sylius\Resource\Generator\RandomnessGeneratorInterface" alias="sylius.random_generator" />
         <service id="Sylius\Component\Resource\Generator\RandomnessGeneratorInterface" alias="sylius.random_generator" />
```

Comment:
Should old `Sylius\Component\Resource\Generator\RandomnessGeneratorInterface` service become a deprecated alias for new one?